src/Counter.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { decrementAction, incrementAction } from './actions/counterActions';
+
+const mapStateToProps = (state) => ({
```

Comment:
tak na przyszłość to możesz sobie dawać to koło connect'a wtedy łatwiej będzie Ci analizować kod

Reply:
w sensie po prostu poniżej tak? :)

Reply:
Tak componentu ;) wtedy łatwiej się analizuje kod ;) ale to jest drobna sprawa :D

Reply:
ok